Enchantments+-/build.gradle

```diff
@@ -20,6 +24,7 @@ dependencies {
     compileOnly "com.github.MilkBowl:VaultAPI:1.7"
     compileOnly 'me.clip:placeholderapi:2.10.9'
     compileOnly files("../Packs/LegacyWrapper.jar")
+    compile 'com.mojang:authlib:1.5.25'
```

Comment:
Isn't authlib included in NMS which would make it better to have compileonly?

Reply:
Yeah that's true...